### ISSUE TITLE
'Failed to set logind session type for session' issue

### ISSUE THREAD

**Issue by CaptainBloodz** [swaywm/sway#5655]
### Please fill out the following:
- **Sway Version:**
- `swaymsg -t get_version` or `sway -v`

sway version 1.5-2c769232 (Sep  2 2020, branch 'HEAD')
command line:
exec env XDG_RUNTIME=/home/user/.cache/sway XDG_RUNTIME_DIR=/home/user/.cache/sway sway -dV > ~/.cache'/sway/sway.log 2>&1

- **Debug Log:**
[sway.log.zip](https://github.com/swaywm/sway/files/5159648/sway.log.zip)

- **Description:**
  - The steps you took in plain English to reproduce the problem.
log reports:

00:00:00.041 [sway/main.c:142] SWAYSOCK=
00:00:00.041 [sway/server.c:43] Preparing Wayland server initialization
00:00:00.048 [ERROR] [backend/session/logind.c:254] Failed to set logind session type for session: Rejected send message, 2 matched rules; type="method_call"
, sender=":1.21" (uid=1000 pid=3079 comm="sway -dV ") interface="org.freedesktop.login1.Session" member="SetType" error name="(unset)" requested_reply="0" de
stination="org.freedesktop.login1" (uid=0 pid=1388 comm="elogind-daemon ")
00:00:00.048 [INFO] [backend/session/logind.c:855] Failed to set logind session type to wayland
00:00:00.048 [INFO] [backend/session/logind.c:858] Successfully loaded logind session
00:00:00.052 [INFO] [backend/backend.c:157] Found 2 GPUs

Sway seems to be working fine anyway...
Thks 4 ur attention, interest & support.

**Comment by tokyovigilante** [comment]
This is expected on systemd <246, but we had a discussion prior to this change being committed that maybe the log wasn't required as it wasn't a fatal error. Happy to submit a patch to quiet this if required, although it is harmless to the session as you suggest.

**Comment by Geo25rey** [comment]
@tokyovigilante Can you share where this previous discussion lies?

**Comment by CaptainBloodz** [comment]
> This is expected on systemd <246, but we had a discussion prior to this change being committed that maybe the log wasn't required as it wasn't a fatal error. Happy to submit a patch to quiet this if required, although it is harmless to the session as you suggest.

NFO: sys-auth/elogind
     Installed versions:  243.7(01:26:41 07/06/2020)(pam policykit -acl -debug -doc -selinux KERNEL="linux")
    As of today no higher version is available on Gentoo.

**Comment by emersion** [comment]
This error message is weird. I think we display something else if the method doesn't exist?

**Comment by tokyovigilante** [comment]
@Geo25rey this is the relevant wlroots [pull request](https://github.com/swaywm/wlroots/pull/2304#discussion_r464601903 and commentary).

@emersion I'd come round to just not logging, as @craigbarnes said it's just best effort, certainly seems more correct to set it on systemd 246+ but if we're on 245 or lower and it can't be set, that's not really a failure or error.

**Comment by emersion** [comment]
Eh, indeed, we don't have a different error message for EINVAL. Yeah, just changing this to a DEBUG message would be fine.
